app/src/main/java/com/example/solanakotlincomposescaffold/usecase/SendTransactionsUseCase.kt

```diff
@@ -37,7 +37,7 @@ object SendTransactionsUseCase {
         JsonRpc20Request(
             method = "sendTransaction",
             params = buildJsonArray {
-                add(Base58.encodeToString(transaction))
+                add(transaction.encodeToBase58String())
```

Comment:
why the change here? if multimult was not working it needs to be fixed 🙏

Reply:
Hm, the import for `Base58` isn't working so that's why I had to use the metaplex lib here